src/selection.js

```diff
@@ -77,7 +69,7 @@ class Selection {
 
     /**
      * Returns an object containing the `row` and `column` current position of the cursor.
-     * @returns {Object}
+     * @returns {import("../ace-internal").Ace.Point}
```

Comment:
Could you convert this to a typedef, and do the same for other instances where same import() is used several times in a file?

Reply:
Yes, sure, but users would need to click twice to see the real type declaration. If this is ok, I would process with changes across codebase